wcfsetup/install/files/lib/system/condition/type/user/AbstractUserIntegerConditionType.class.php

```diff
@@ -84,8 +86,39 @@ public function matches(object $object): bool
     /**
      * @return string[]
      */
-    private function getConditions(): array
+    protected function getConditions(): array
     {
         return ["=", ">", "<", ">=", "<="];
     }
+
+    #[\Override]
+    public function canMigrateConditionData(string $objectType): bool
+    {
+        return $this->migrateConditionObjectType !== null && $objectType === $this->migrateConditionObjectType;
```

Comment:
```suggestion
        return $objectType === $this->migrateConditionObjectType;
```

`$objectType` can never be `null` thus a strict equality check will always fail when `$this->migrateConditionObjectType` is `null`.

Also, why does `null` prevent a migration here but allows it for `AbstractUserBooleanConditionType`?